xla/stream_executor/rocm/rocm_executor.cc

```diff
@@ -400,6 +404,18 @@ std::string GetPCIBusID(hipDevice_t device) {
   return pci_bus_id;
 }
 
+bool IsEccEnabled(hipDevice_t device, bool* result) {
```

Comment:
Can you make this return a `absl::StatusOr<bool>` (so the `bool` should be the result) and move the logging of the error into `CreateDeviceDescription`?